be/src/storage/rowset/beta_rowset_writer.cpp

```diff
@@ -159,6 +159,12 @@ Status BetaRowsetWriter::flush_src_rssids(uint32_t segment_id) {
     return Status::OK();
 }
 
+// these files will be cleaned up with error handling by the GC background even though they have not been deleted here after an error occurred, so we just go ahead here and print the error message only when we encounter an error except Status::NotFound().
+void BetaRowsetWriter::try_delete_file(const std::string& path) {
+    auto st = _fs->delete_file(path);
+    LOG_IF(WARNING, !(st.ok() || st.is_not_found())) << "Fail to delete file=" << path << ": " << st.to_string();
+}
+
 HorizontalBetaRowsetWriter::HorizontalBetaRowsetWriter(const RowsetWriterContext& context)
```

Comment:
This will hide code line number and calling method names in LOG, make it harder to debug